roles/create-devstack-local-conf/tasks/main.yml

```diff
@@ -190,16 +190,8 @@
     cmd: |
       set -e
       set -x
-      # enable q-lbaasv2 and disable q-lbaas for stable/mitaka branch
-      if [[ "{{ global_env.OS_BRANCH }}" == "stable/mitaka" ]];then
-          services='octavia,o-cw,o-hm,o-hk,o-api,q-lbaasv2'
-          cat << EOF >> /tmp/dg-local.conf
-          disable_service q-lbaas
-          enable_plugin neutron-lbaas https://opendev.org/openstack/neutron-lbaas
-      EOF
-      else
-          services='octavia,o-cw,o-hm,o-hk,o-api'
-      fi
+
+      services='octavia,o-cw,o-hm,o-hk,o-api'
```

Comment:
Please add echo message to show octavia is not available for some OpenStack releases in testing log, somebody maybe feel strange why they enable `octavia` in playbook, but in actually octavia can't be install.

Reply:
I don't see any updates about output message I mention in my 3 days ago comment

Reply:
if this task skiped by condition, it will not actually run, so `echo` message cannot work. maybe following is better:
`- name: create devstack local conf with octavia enabled for {{ global_env.OS_BRANCH }} release(M, N, Q unsupported)`

Reply:
OK, thank you.